src/librustdoc/html/static/main.js

```diff
@@ -1562,14 +1562,29 @@
     window.initSidebarItems = initSidebarItems;
 
     window.register_implementors = function(imp) {
-        var list = document.getElementById('implementors-list');
+        var implementors = document.getElementById('implementors-list');
+        var synthetic_implementors = document.getElementById('synthetic-implementors-list');
         var libs = Object.getOwnPropertyNames(imp);
         for (var i = 0; i < libs.length; ++i) {
             if (libs[i] === currentCrate) { continue; }
             var structs = imp[libs[i]];
             for (var j = 0; j < structs.length; ++j) {
+                var struct = structs[j];
+                var list = struct.synthetic ? synthetic_implementors : implementors;
+
+                var bail = false;
+                for (var k = 0; k < struct.types.length; k++) {
+                    if (window.inlined_types.has(struct.types[k])) {
+                        bail = true;
+                        break;
+                    }
+                }
+                if (bail) {
+                    continue;
```

Comment:
Instead of setting `bail` to `true`, you could just add a label on your `for` loop and then `continue 'the_loop`.

Reply:
I didn't even know that js had labeled loops :) I'll make the change.